contracts/fee-manager/.editorconfig

```diff
@@ -0,0 +1,11 @@
+root = true
```

Comment:
Does VSCode use this file or can we remove it?

Reply:
I think it's a default part of the template? A cargo thing maybe? Not sure.